dashboard/src/features/container/routes.js

```diff
@@ -0,0 +1,20 @@
+import Container from './components/Container'
+
+import { getAuthenticationStatus } from 'features/authn/actions'
+import { fetchCoreInfo } from 'features/core/actions'
+import { fetchTestnetInfo } from 'features/testnet/actions'
+
+const CORE_POLLING_TIME = 2 * 1000
+const TESTNET_INFO_POLLING_TIME = 30 * 1000
+
+export default (store) => ({
+  path: '/',
+  component: Container,
+  onEnter: () => {
```

Comment:
Just to clarify, this only ever gets called once for the lifetime of the whole app, right? If you want this logic to run again, you have to reload the page?